main.go

```diff
@@ -164,7 +167,37 @@ func (s *admissionWebhookServer) createVolumesPatch(p string, volumes []corev1.V
 	return jsonpatch.NewOperation("add", path.Join(p, "volumes"), volumes)
 }
 
+func parseResources(v string, logger *zap.SugaredLogger) map[string]int {
+	var nsmURLs []*nsurl.NSURL
+	poolResources := make(map[string]int)
+
+	for _, rawNetNS := range strings.Split(v, ",") {
+		netNS, err := url.Parse(rawNetNS)
+
+		if err != nil {
+			logger.Errorf("Malformed NS annotation: %+v", rawNetNS)
+			return nil
+		}
+		nsmURLs = append(nsmURLs, (*nsurl.NSURL)(netNS))
+	}
+
+	for _, nsmURL := range nsmURLs {
+		labels := nsmURL.Labels()
+		if _, ok := labels["sriovToken"]; ok {
+			interfacePools := strings.Split(labels["sriovToken"], ",")
+			if _, ok := poolResources[interfacePools[0]]; !ok {
+				poolResources[interfacePools[0]] = 1
+				continue
+			}
```

Comment:
Note: default int value is zero (i.e. for non-exsist keys we always have zero). So we can just remove this condition.
```suggestion
```

Reply:
Done